packages/server/graphql/public/typeDefs/FeatureFlagScope.graphql

```diff
@@ -0,0 +1,8 @@
+"""
+The list of scopes available for feature flags
+"""
+enum FeatureFlagScope {
```

Comment:
+1 can remove?

Reply:
We need this for the `addFeatureFlag` mutation